template/readme/local-development.md

```diff
@@ -80,6 +80,15 @@ There are also other changes you can make if you choose to match the Acquia Clou
 
 Once you've made these changes and completed the steps in Drupal VM's Quick Start Guide, you may run `vagrant up` to bring up your local development environment, and then access the site via the configured `drupal_domain`.
 
+### Using DrupalVM 3.0
+
+This project has been configured to run with [DrupalVM 3.0](geerlingguy/drupal-vm). You can call `vagrant up` directly from the project root as of version 3.0. The new DrupalVM is pulled in as a dev dependency through composer, and uses a delegating Vagrantfile to call the project's Vagrantfile in the `vendor/geerlingguy/drupal-vm` folder. The project is pre-configured with a config.yml file located at /config/drupalvm/config.yml. You may use a local.config.yml file to overwrite any values in the repositories copy of the file.
+
+It is recommended you have the following set up on your local machine.
```

Comment:
I don't like making people depend on `vagrant-auto_network`, as I've had little issues with it here or there, and it is not always consistent with new version of Vagrant/across OSes. It would possibly be better to leave the default Drupal VM IP setup, and recommend it (and a change to the IP address in a dev's `local.config.yml` file) iff the dev is comfortable installing the extra plugin.